### PR TITLE
groups: s3 css fixes

### PR DESCRIPTION
**Pull request by rcrdlbl**
resolves everything in #1160.

### CODE REVIEW

**Review by patosullivan — APPROVED** [review]
LGTM

File: ui/src/profiles/ProfileCoverImage.tsx [hunk 1]
```diff
@@ -21,13 +21,12 @@ const emptyContact: Contact = {
 };
 
 export default function ProfileCoverImage({
-  ship,
+  cover,
   className,
   children,
 }: CoverProps) {
-  const contact = useContact(ship);
+  // const contact = useContact(ship);
```
Comment (jamesacklin) [inline ui/src/profiles/ProfileCoverImage.tsx:28]: Delete if unused?

Comment (rcrdlbl) [reply ui/src/profiles/ProfileCoverImage.tsx:28]: you know what? you're right.

**Comment by jamesacklin** [comment]
Nice, thank you

**Pull request merged**